google_compute_engine/metadata_scripts/script_retriever.py

```diff
@@ -50,29 +56,38 @@ def _DownloadGsUrl(self, url, dest_dir):
     Returns:
       string, the path to the file storing the metadata script.
     """
-    try:
-      subprocess.check_call(
-          ['which', 'gsutil'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    except subprocess.CalledProcessError:
-      self.logger.warning(
-          'gsutil is not installed, cannot download items from Google Storage.')
-      return None
-
     dest_file = tempfile.NamedTemporaryFile(dir=dest_dir, delete=False)
     dest_file.close()
     dest = dest_file.name
 
-    self.logger.info('Downloading url from %s to %s using gsutil.', url, dest)
+    self.logger.info(
+        'Downloading url from %s to %s using authentication token.', url, dest)
+
+    if not self.token:
+      response = self.watcher.GetMetadata(
+          self.token_metadata_key, recursive=False, retry=False)
+
+      if not response:
+        self.logger.warning(
```

Comment:
This seems like a non-exceptional case. Perhaps it should be a standard logger.info?

```
self.logger.info('Authentication token not found. Attempting unauthenticated download.')
```